Revamp the ESLint setup, closes LEA-1866 (#765)
leather-io/mono: packages/eslint-config/README.md

## Setup

### Inside @leather.io/mono

You don’t need to install or link this package within monorepo packages. The base config 

> will be applied to all packages and apps by default. To add custom configurations (e.g., React) for a new 

Reviewer: ```suggestion
is applied to all packages and apps by default. To add custom configurations (e.g., React) for a new 
```

active voice
